tests/test_execute.py

```diff
@@ -243,47 +290,50 @@ def test_execute_notebook_save_profiling_data(
     [
         (
             "abc",
-            "Invalid save_profiling_data, path must end with .csv",
+            "Invalid save_profiling_data(.*), path must end with .csv",
```

Comment:
nit: 
If this is shown in multiple places you can make it as constant variable
```python
INVALID_SAVE_PROFILING_PATH_MSG ="Invalid save_profiling_data(.*), path must end with .csv"
```

Reply:
Done!